src/tests/ftest/scrubber/aggregation.py

```diff
@@ -0,0 +1,56 @@
+"""
+  (C) Copyright 2018-2023 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import time
+
+from scrubber_test_base import TestWithScrubber
+
+
+class TestScrubberEvictWithAggregation(TestWithScrubber):
+    # pylint: disable=too-many-nested-blocks
```

Comment:
Is this pylint needed? It doesn't seem so

Reply:
Need to check... I was getting some pylint errors on one of my PR...

Reply:
Done.